action.yml

```
@@ -152,6 +152,38 @@ runs:
         GITHUB_REPOSITORY: ${{ github.repository }}
         GITHUB_SHA: ${{ github.sha }}
       run: |
+        # Work out if release should be marked as the latest
+        # Only do the for stable semver tags
+        # Note = not using "legacy" as GitHub seems to only consider if it's the latest tag in a particular major line
+        # whereas we only want to mark it the latest if it's the latest tag overall
+        MAKE_LATEST="false"
+        if [[ $TAG =~ ([0-9]+)\.([0-9]+)\.([0-9]+) ]]; then
+          # Gets all tags for all majors from via GitHub API
+          # https://docs.github.com/en/rest/git/tags?apiVersion=2022-11-28
+          RESP_CODE=$(curl -w %{http_code} -s -o __response.json \
+          -X GET "https://api.github.com/repos/${GITHUB_REPOSITORY}/tags?per_page=100" \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer ${{ github.token }}" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          )
+          if [[ $RESP_CODE != "200" ]]; then
+            echo "Unable to read list of tags - HTTP response code was $RESP_CODE"
+            exit 1
+          fi
+          # Get the latest stable tag
+          LATEST_TAG=$(jq -r '.[].name' __response.json | grep -Po '^[0-9]+\.[0-9]+\.[0-9]+$' | sort -V -r | head -n 1)
+          echo "LATEST_TAG is $LATEST_TAG"
+          if [[ $LATEST_TAG =~ ([0-9]+)\.([0-9]+)\.([0-9]+) ]]; then
```

```suggestion
          if [[ $LATEST_TAG =~ ^([0-9]+)\.([0-9]+)\.([0-9]+)$ ]]; then
```
We may not need to be that strict here but since we're doing regex rather than just checking for "a value", we may as well use the strict regex.

---

Updated